src/components/Dashboard/Dashboard.tsx

```diff
@@ -52,21 +55,25 @@ export const Dashboard = () => {
         <ul className={style.activities}>
           {activities?.map(({ timesDone, id, name }) => (
             <div
-              className={timesDone === 0 ? style.activityZero : style.activity}
+              className={classnames(style.activity, {
+                [style.zero]: timesDone === 0,
+                [style.more]: timesDone > 0,
+              })}
               key={id}
             >
               <span className={style.timesDone}>{timesDone}</span>
-              <li>{name}</li>
-
+              <li className={style.nameOfActivity}>{name}</li>
               <button
-                className={style.btnDone}
+                className={classnames(style.buttonDone, {
+                  [style.zero]: timesDone === 0,
+                  [style.more]: timesDone > 0,
+                })}
+                aria-label="activity done"
```

Comment:
```suggestion
                aria-label={`Aktiva ${name} hotova`}
                aria-description={`Aktivitu ${name} hotova ${timesDone}`}
```

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description